Makefile

```diff
@@ -41,6 +41,8 @@ install: go.sum
 mod:
 	@echo "--> Updating go.mod"
 	@go mod tidy
+	@echo "--> Updating go.mod in ./test/testground"
+	@(cd ./test/testground && go mod tidy)
```

Comment:
👍

Reply:
do we want a `go work sync` prior to both of these commands?

Reply:
Yeah makes sense

Reply:
updated in https://github.com/celestiaorg/celestia-app/pull/2979/commits/ac5bde2528b3c25110288fe98ed38262970a88b1